src/main/kotlin/uk/gov/justice/digital/hmpps/manageusersapi/adapter/external/UserApiService.kt

```diff
@@ -17,6 +17,11 @@ class UserApiService(
     val log: Logger = LoggerFactory.getLogger(this::class.java)
   }
 
+  fun updateUserEmailAddressAndUsername(userId: UUID, username: String, email: String) =
+    externalUsersWebClientUtils.put("/users/id/$userId/email", mapOf("username" to username, "email" to email))
+
+  fun hasPassword(userId: UUID) = externalUsersWebClientUtils.get("/users/$userId/password/present", Boolean::class.java)
```

Comment:
You will need my latest commit for this to work correctly - I think the external users endpoint for this should also have a role

Reply:
See comment above?

Reply:
Also discussed with @karenmillermoj and verified permissions in external-users-api all align with user permissions on original Auth end point, so OK